packages/backend/src/apps/vault-workspace/common/update-table-row.ts

```diff
@@ -4,14 +4,22 @@ const updateTableRow = async (
   $: IGlobalVariable,
   vaultId: string,
   update: { [key: string]: string }, // column name: value
-): Promise<void> => {
+): Promise<{
+  [key: string]: any & {
+    _metadata: {
+      success: boolean
+      rowsUpdated: number
+    }
+  }
+}> => {
   const res = await $.http.put('/api/tables/row', { id: vaultId, update })
-  $.setActionItem({
-    raw: {
+  return {
+    _metadata: {
```

Comment:
Comment: Was a little bit worried about us breaking backward compatibility (the lodash get path for the `success` variable will change from `{{step.abc-def.success}}` to `{{step.abc-def._metadata.success}}`).

But i checked the DB, looks like no one is using this var so we're good.

Reply:
the reason why no one was using this was because there wont be a `success=false` result, it will just error out

Reply:
but yeah good catch

Reply:
oh ya true ah. XD